src/libraries/System.Transactions.Local/src/System/Transactions/TransactionManager.cs

```diff
@@ -310,6 +310,31 @@ public static TimeSpan DefaultTimeout
                 }
                 return s_defaultTimeout;
             }
+            set
+            {
+                TransactionsEtwProvider etwLog = TransactionsEtwProvider.Log;
+                if (etwLog.IsEnabled())
+                {
+                    etwLog.MethodEnter(TraceSourceType.TraceSourceBase, "TransactionManager.set_DefaultTimeout");
+                }
+
+                s_defaultTimeout = ValidateTimeout(value);
+
+                if (s_defaultTimeout != DefaultSettingsSection.Timeout)
+                {
+                    if (etwLog.IsEnabled())
+                    {
+                        etwLog.ConfiguredDefaultTimeoutAdjusted();
+                    }
+                }
+
+                s_defaultTimeoutValidated = true;
```

Comment:
In DefaultTimeout's **Get** method, I see if the value is being accessed the first time, it will validate and adjust to make sure value is not exceeding MaxTimeout and then set variable of s_defaultTimeoutValidated to true which result in no more check in further Get. Is this intended behavior, should DefaultTimeout be always less than MaxTimeout?  It leads to test result below (set MaxTimeout multiple times in order) after enabling setter of the properties:

>Set MaximumTimeout to 00:00:30
  -Default:00:00:30
  -Maximum:00:00:30

>Set MaximumTimeout to 00:00:25
  **-Default:00:00:30
  -Maximum:00:00:25**